artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/broker.xml

```diff
@@ -61,10 +61,13 @@ ${ping-config.settings}${journal-buffer.settings}${connector-config.settings}
       <global-max-size>100Mb</global-max-size>
 
       <acceptors>
+
          <!-- useEpoll means: it will use Netty epoll if you are on a system (Linux) that supports it -->
+         <!-- amqpCredits: The number of credits sent to AMQP producers -->
+         <!-- amqpLowCredits: The server will send the # credits specified at amqpCredits at this low mark -->
```

Comment:
Should there maybe be some coverage of these options in the documentation? Do we think enough people will tweak them that they should be in the default configuration file?